nextcloud/android
# Bugfix E2E File Deletion For V1

**How to Test**

1. Create encrypted folder
2. Upload multiple files inside encrypted folder
3. Delete one file from encrypted folder
4. Refresh folder, file must be deleted
5. Try to add some file again, file conflict notification should't appear

Test it for V1 (Nextcloud 27.1.7) and V2 (v28 and above)

### Note

E2E version must be lower than 2.0
